tools/accuracy_checker/accuracy_checker/config/config_reader.py

```diff
@@ -683,6 +683,10 @@ def process_dataset(datasets_configs):
             if annotation_conversion_config:
                 command_line_conversion = (create_command_line_mapping(annotation_conversion_config,
                                                                        'source', ANNOTATION_CONVERSION_PATHS))
+                command_line_mapping = {key: args[value] for key, value in command_line_conversion.items()}
+                datasets_config.update({
+                    'command_line_mapping': command_line_mapping
```

Comment:
better to use under score for parameter name for emphasis that it is private info, automatically generated, can not be specified by user
```suggestion
                    '_command_line_mapping': command_line_mapping
```

Reply:
Renamed